File: ABC347A/abc347a.cs
using System;

class Program {
    static void Main() {
        int N, K;
        string input = Console.ReadLine();
        N = int.Parse(input.Split(' ')[0]);
        K = int.Parse(input.Split(' ')[1]);
        input = Console.ReadLine();
        int[] A = input.Split(' ').Select(int.Parse).ToArray();
        for (int i=0; i<N; ++i) {
            if (A[i] % K == 0)
                Console.WriteLine(A[i]/K);
        }
    }
}

File: ABC347A/abc347a.dart
import 'Dart:io';
void main() {
    List nk = stdin.readLineSync()!.split(" ").map((x) => int.parse(x)).toList();
    List a = stdin.readLineSync()!.split(" ").map((x) => int.parse(x)).toList();
    int n = nk[0], k = nk[1];
    for (int i=0; i<n; ++i) if (a[i] % k == 0) print(a[i] ~/ k);
}
